### ISSUE TITLE
Initial suggestions for plugin release

### ISSUE THREAD

**Issue by misraved** [francois2metz/steampipe-plugin-freshping#2]
Thanks @francois2metz for another new plugin 👍. I had an initial look at it and did test out the tables which seem to be working great 👍. The overall structure looks good and I believe it is almost release ready 😄 . 

I just have a few minor suggestions post which we could move the plugin to the hub:
- The example queries are good but a  bit light, but the API does not return a lot of columns either. Could you please add `;` at the end of queries, both in the table docs and docs/index.md file? For instance https://github.com/francois2metz/steampipe-plugin-freshping/blob/main/docs/tables/freshping_check.md#list-checks-in-failure is missing `;` at the end of the query.

- In the `config/freshping.spc` file I think it would be valuable to mention about the environment variables, you could use the format mentioned in https://github.com/turbot/steampipe-plugin-circleci/blob/main/config/circleci.spc#L5.

- As per the API documents, I think we should rename the authentication token to use `api_key` instead of `key`. Likewise, the environment variables should be updated to use `FRESHPING_API_KEY` instead of `FRESHPING_KEY`.

Reference:

```
  Authentication

Authentication is done over Basic Authentication using your organization API key as Username and Freshping subdomain as Password to authenticate the request.

curl -v -u <apikey>:<freshping_subdomain> -H "Content-Type: application/json" -X GET 'https://api.freshping.io/api/v1/'

For example, if your API key is abcdefghij1234567890 and Freshping URL is xyz.freshping.io, the curl command to use is:

curl -v -u abcdefghij1234567890:xyz -H "Content-Type: application/json" -X GET 'https://api.freshping.io/api/v1/checks/'
```

- In the `docs/index.md` file could we add the credential section as mentioned in https://github.com/turbot/steampipe-plugin-circleci/blob/main/docs/index.md#credentials. This will help us to maintain a consistent format across all the plugins and also provide information about the `api_key` on a single page.

- Could we move the info about the env variables to a new section as mentioned in https://github.com/turbot/steampipe-plugin-circleci/blob/main/docs/index.md#credentials-from-environment-variables? I think this would be particularly useful for beginners in terms of using environment variables.

- The brand_color parameter in `docs/index.md` should be updated to use `#F28C28`. I will provide more information once the social graphics icon has been created.

- I couldn't find any instances of pagination in the API docs. Is it handled internally by the SDK?

- I did observe that the APIs are prone to rate limits. Should we add a retry function to make sure that throttling is handled?

Overall I must admit that the plugin is perfect. Once the minor feedback comments are addressed we could push for a release 👍 .

Please let us know if you have any questions on the feedback comments 😄 . Happy to help 👍 .

**Comment by misraved** [comment]
@francois2metz The brand_color is correct (`1F9999`). Please ignore the comment on updating the brand colour. Apologies for the churn 👍 .

**Comment by francois2metz** [comment]
I have done most of the points, except the last ones
>     * I couldn't find any instances of pagination in the API docs. Is it handled internally by the SDK?

I could find anything on it. I asked the support about it without any answers. I also created a lot of checks to see it could reach the pagination point without success.

>     * I did observe that the APIs are prone to rate limits. Should we add a retry function to make sure that throttling is handled?

I forgot to handle that.

**Comment by misraved** [comment]
Thanks for making the changes @francois2metz. I believe once the rate limits are handled, we could push for a plugin release 👍.

Great work on the new plugin 🎉 .

**Comment by francois2metz** [comment]
@misraved I added a retry function and released the plugin a 0.3.0.
